### ISSUE TITLE
Talkback screen reader focus goes away on toggle switches

### ISSUE THREAD

**Issue by silverli** [tmobile/magentaA11y#51]
Conversation in this thread: https://tmus-digital.slack.com/archives/C048XPT06JY/p1689112527046689

This looks to be an issue relating to CSS and maybe some use of overflow:hidden or something similar. No focus appears on the Toggle Switches when Talkback is in use. See video below:

https://github.com/tmobile/magentaA11y/assets/12848168/78ccc439-69a5-48c6-bd8e-0c715a830419
